src/pal/tools/gen-buildsys-win.bat

```diff
@@ -19,10 +19,13 @@ set __SourceDir=%1
 set __VSVersion=%2
 set __Arch=%3
 set __CmakeGenerator=Visual Studio
+
+if /i "%__VSVersion%" == "vs2019" (set __CmakeGenerator=%__CmakeGenerator% 16 2019)
 if /i "%__VSVersion%" == "vs2017" (set __CmakeGenerator=%__CmakeGenerator% 15 2017)
 if /i "%__VSVersion%" == "vs2015" (set __CmakeGenerator=%__CmakeGenerator% 14 2015)
-if /i "%__Arch%" == "x64" (set __CmakeGenerator=%__CmakeGenerator% Win64)
-if /i "%__Arch%" == "arm" (set __CmakeGenerator=%__CmakeGenerator% ARM)
+
+if /i "%__Arch%" == "x64" (set __ExtraCmakeParams=%__ExtraCmakeParams% -A x64)
```

Comment:
Why was it changed?

Reply:
Same as for the CoreFX PR: https://github.com/dotnet/corefx/pull/35197#discussion_r255655713

The VS 2019 generator removed the legacy target specific generator names (which have been deprecated since CMake 3.2) and requires the `-A` option. It is easier to just use `-A` everywhere